content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -79,10 +83,12 @@ const NewTabPage = React.createClass({
           </section>
 
           <section>
-            <Spotlight page={PAGE_NAME} showRating={props.Spotlight.metadataRating} sites={props.Spotlight.rows} />
+            <Spotlight page={PAGE_NAME} length={spotlightLength}
+              showRating={props.Spotlight.metadataRating}
+              sites={props.Spotlight.rows} />
           </section>
 
-          {this._renderRecentActivity()}
+          { this.renderRecentActivity() }
```

Comment:
Turns out the JSX grammar wants an expression, not a statement, so I ended up with this:

```javascript
 { props.Spotlight.weightedHighlights ? null : this.renderRecentActivity() }
```